src/index.js

```diff
@@ -368,7 +367,7 @@ class MiniCssExtractPlugin {
                   'linkTag.href = fullhref;',
                   crossOriginLoading
                     ? Template.asString([
-                        `if (linkTag.href.indexOf(window.location.origin + '/') !== 0) {`,
+                        `if (linkTag.href.indexOf(location.origin + '/') !== 0) {`,
```

Comment:
Always need `window`, you can have scope problem in some rare cases

Reply:
What scope problem are you referring to?

`var location = 1` or `location=1` in global scope redirect the page, as does `window.location=1` - it's exceptionally unlikely the value would be redefined. It could be shadowed, but that's equally true for any variable including `window`.

Reply:
Note that the code already uses `document` rather than `window.document`.

Reply:
Fair point... we should side with implicit or explicit conventions throughout

Reply:
I’ve had scope issues before. Rarely but they do happen. Even if wasted bytes. This plugin gets like 5mil downloads a week. Eventually complaints would roll in. We are talking race conditions. 

Stupid question, what if we declared window as a variable, then w.document or w.location? One way to save bytes maybe. Can’t speak if race conditions would come of this never tried

Reply:
> We are talking race conditions.

I don't see a race condition here.

> what if we declared window as a variable, then w.document or w.location? One way to save bytes maybe.

If gzip's involved, that uses more bytes than `window.document`.

Fwiw I think omitting `window` is fine.